backend/app/models/collection_job.py

```diff
@@ -79,13 +88,26 @@ class CollectionJobUpdate(SQLModel):
     trace_id: str | None = None
 
 
-class CollectionJobPublic(SQLModel):
-    id: UUID
-    action_type: CollectionActionType
-    collection_id: UUID | None = None
+##Response models
+class CollectionJobBasePublic(SQLModel):
+    job_id: UUID
     status: CollectionJobStatus
     error_message: str | None = None
-    inserted_at: datetime
-    updated_at: datetime
 
-    collection: CollectionPublic | None = None
+    model_config = ConfigDict(
+        from_attributes=True,
+        exclude_none=True,
+    )
+
+
+class CollectionJobImmediatePublic(CollectionJobBasePublic):
```

Comment:
this is bieng return in collection/create/delete endpoint
You can simply put this in models/collection.py
And 
```suggestion
class CollectionJobInfo(SQLMODEL):
       id:
       inserted_at
       updated_at
       status

class CollectionJobResponse(SQLMODEL):
       job: CollectionJobInfo
```
and leaving room for whatever extra info we want to put in create/delete

This will clean up lots of code here and have a logical sepration

Reply:
Since they are collection job object, it makes much more sense to me to keep the response models in models/collection_job.py, even if they are being returned in collection endpoint

Reply:
While what you are suggesting here with "collectionjobinfo" is a really clean way of handling response bodies, making these changes would also require some consideration (not much but some), additionally how we represent job here would affect jobs in all job related modules